### PR TITLE
test: modify Java test code

### PR DESCRIPTION
**Pull request by hyeok7777**
- Fixed some variable names in file ConfigUtilsTest.java that do not conform to the Java naming convention
- Remove unnecessary public

### CODE REVIEW

**Comment by junoyoon** [comment]
@hyeok7777 
Thx for contribution.
One thing I noticed is that blarBlar_BlarBlar is out internal convention(undocumented yet) to describe the test condition.
Please make test method name keep this convention plz

**Review by junoyoon — CHANGES_REQUESTED** [review]
Thx for contribution.
One thing I noticed is that blarBlar_BlarBlar is out internal convention(undocumented yet) to describe the test condition.
Please make test method name keep this convention plz

**Comment by junoyoon** [comment]
@hyeok7777 please check my comment

File: scavenger-agent-java/src/test/java/com/navercorp/scavenger/javaagent/util/ConfigUtilsTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void setProperties() {
 
             @Test
             @DisplayName("it expands variables with braces")
-            public void expandVariables_with_braces() {
+            void expandVariablesWithBraces() {
```
Comment (junoyoon) [inline scavenger-agent-java/src/test/java/com/navercorp/scavenger/javaagent/util/ConfigUtilsTest.java:77]:
```suggestion
            void expandVariables_with_braces() {
```

**Comment by hyeok7777** [comment]
@junoyoon 
Forgive me for being late. I modified and committed what you said.

**Review by taeyeon-Kim — APPROVED** [review]
thx

**Review by kojandy — APPROVED** [review]

**Comment by junoyoon** [comment]
이번 건은 외부 fork 로 부터 PR 이라.. merge commit 이 들어왔습니다. 
어쩔수 없이, 이번엔 허용합니다.

**Pull request merged**